### ISSUE TITLE
higher battery consumption by binder_2

### ISSUE THREAD

**Issue by KamoCode** [ubports/ubuntu-touch#1506]
- Device: FP2
- Channel: stable (2019 jan to nov, release-candidate (from november until now)
- Build: 2020-W26

### Steps to reproduce
To observe an increased CPU load with binder_2 you only have to restart or reboot your phone. It happens so often that it should not take more than 3 tries. Check it with top in terminal. To get back a normally load sometimes I restart the phone more than 3 times to get a normal case. I believe that the best way to get a normal load is to shutdown the device, than start and finally reboot, but it could also be my delusion. 

### Expected behavior
In normally cases binder_2 is running with 1,7-2,3% CPU load. I would say it is much too, but for the FP2 it seems to be normal. If the device have a happy day binder_2 runs with 1,3%.

### Actual behavior
The bad case is that the binder_2 CPU load is going up to 12% or sometimes more. That happens a few time (1-2minutes) after a system start. After binder_2 was started it swings at about 2-5% and then it slowly goes higher and higher. If this happens the phone is going slightly warmer on the CPU position and the battery goes empty in less one day. By a normal CPU load my battery needs to be recharged after around 3 to 4 days.

### Logfiles and additional information
I found no records of binder_2 in the logs.

**Comment by UniversalSuperBox** [comment]
Hi @KamoCode,

Is this device running Anbox currently?

**Comment by UniversalSuperBox** [comment]
Hi @KamoCode,

Since we haven't heard from you in a while, I'll close this issue report. Please let us know if this issue recurs.

**Comment by KamoCode** [comment]
Hi @UniversalSuperBox,

I'm so sorry for the late answer, but I'm back now.
Yes, my device is running anbox, but normally it is shut off... through the battery^^.
For the chronological sequence, I have installed ubuntu-touch on this phone and around 2-3month later I have perceived that sometimes the battery consumption is very high. I look for the problem an only find binder_2 as a correlation. After that I change to the release candidate, install a Libertine container and few weeks later anbox. All these doesn't change anything relating to the described problem above.
